src/Check/Resource/Http/StatusCheck.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Leankoala\HealthFoundation\Check\Resource\Http;
```

Comment:
Leankoala\HealthFoundation\Check\Http

I think there could be a http group on its own.

Reply:
Yes, we can pull the protocol level one up. It makes sense if you also want to do content checks.